src/components/Reports/TotalReport/TotalProjectReport.jsx

```diff
@@ -287,7 +242,7 @@ function TotalProjectReport(props) {
 
   return (
     <div>
-      {dataLoading ? (
+      {!dataReady ? (
         <Loading align="center" darkMode={darkMode}/>
```

Comment:
The Loading animation can be centered Vertically around the mid area point of the div bar. It works correctly around the X-axis.